cloud/pom.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Oh, just noticed this -- should this perhaps be in a directory `hadoop-cloud` to match the profile and module name?